monai/handlers/tensorboard_handlers.py

```
@@ -50,6 +50,11 @@ class TensorBoardHandler:
 
     def __init__(self, summary_writer: SummaryWriter | SummaryWriterX | None = None, log_dir: str = "./runs"):
         if summary_writer is None:
+            if SummaryWriter is None:
+                raise RuntimeError(
+                    "TensorBoardHandler requires tensorboard to be installed. "
+                    "Please install it with: pip install tensorboard"
+                )
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)